extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_dropdown.php

```diff
@@ -0,0 +1,24 @@
+<div class="wds-global-navigation__<?= Sanitizer::escapeClass( $type ); ?> wds-dropdown wds-is-active">
+	<div class="wds-dropdown-toggle wds-global-navigation__dropdown-toggle">
+		<?php if ( $model['header']['type'] === 'line-text' ): ?>
+			<span><?= DesignSystemHelper::renderText( $model['header']['title'] ) ?></span>
+		<?php elseif ( $model['header']['type'] === 'avatar' ): ?>
+			<?= Html::element( 'img', [
+				'class' => 'wds-avatar wds-is-circle',
+				'src' => $model['header']['url'],
+				'alt' => $model['header']['username']['value']
+			] ); ?>
+		<?php endif; ?>
+		<?= DesignSystemHelper::getSvg(
+			'wds-icons-dropdown-tiny',
+			'wds-icon wds-icon-tiny wds-dropdown-toggle-chevron'
+		) ?>
+	</div>
+	<div class="wds-dropdown-content wds-global-navigation__dropdown <?= $rightAligned ? 'wds-is-right-aligned' : ''; ?>">
```

Comment:
These class names will have to be updated. @tomasznapieralski is changing them in DS to be:
```
wds-dropdown__content wds-global-navigation__dropdown-content
```